### ISSUE TITLE
Click on notification action does not hide notifications drawer (Android)

### ISSUE THREAD

**Issue by skaldo** [notifee/react-native-notifee#200]
Hello,
I noticed, that the click on notification action configured as follows does properly start the app and trigger event, but it does not hide/dismiss notification drawer.

```
notifee.displayNotification({
  title: 'Title',
  body: 'Body',
  android: {
    channelId: 'default',
    actions: [
      {
        title: 'Close notification drawer please',
        pressAction: {
          id: 'doFancyStuff',
          launchActivity: 'default'
        },
      },
    ],
  },
});
```

In event handler of type `EventType.ACTION_PRESS` I do dismiss notification by `notifee.cancelNotification`, if that helps.

I assume, that Notifee does not call broadcast `android.intent.action.CLOSE_SYSTEM_DIALOGS` under the hood.

Notifee version: 0.15.0

**Comment by skaldo** [comment]
For the ones, that are interested, I do have a workaround.
Simply added a method `NotifeeReactUtils.hideStatusBar();` and called it in `public void onNotificationEvent(NotificationEvent notificationEvent)`.

Here is a patch: https://gist.github.com/skaldo/02d053e986b6081374fa5eec6cda3ec1

**Comment by helenaford** [comment]
Thanks for the notes. Will put this under review. Looks like there's 2 things; one there's a potential issue that we're not calling that broadcast to clear the drawer, and two a possible enhancement.

**Comment by NicolasSmit** [comment]
We are also facing the same issue, do you any idea on the fix release ?

**Comment by helenaford** [comment]
Included in latest release  (v1.5.0) `notifee.hideNotificationDrawer()`. @skaldo thanks for the solution.
